configs/setups/awiesm/awiesm-2.1.yaml

```diff
@@ -210,6 +211,10 @@ fesom:
                 "2.1":
                         version: "2.1"
                         ALE_scheme: 'linfs' # NOTE(PG): This should probably be overriden in the AWIESM 2.x yaml or in the user runscript.
+                "2.1-paleodyn-gamma":
+                        # FIXME(PG): ....does that make sense? Choose version sets the version???
```

Comment:
I think it makes sense for the following reason: in `AWIESM` the setup version is strictly linked to the model version. Therefore, there is no point for the user to manually specify the model version one by one, instead the user can simply select the `general.version` and then this file takes care of selecting the desired version of the models included.